modules/issue_tracker/php/uploadhelper.class.inc

```diff
@@ -0,0 +1,209 @@
+<?php declare(strict_types=1);
+/**
```

Comment:
@johnsaigle can you review this?

Reply:
I commented in my October 29th review that I think this module should use the new File Upload handler, as well as many other general concerns I had.